dot/sync/chain_sync.go

```diff
@@ -46,6 +48,20 @@ const (
 	tip
 )
 
+func chainSyncStateString(mode chainSyncState) string {
+	switch mode {
+	case bootstrap:
+		return "bootstrap"
+	case tip:
+		return "tip"
+	default:
+		return "unknown"
+	}
+}
```

Comment:
```suggestion
func (css chainSyncState) String() {
	switch css {
	case bootstrap:
		return "bootstrap"
	case tip:
		return "tip"
	default:
		return "unknown"
	}
}
```
The logger will call this method when using `%s`.